underfs/hdfs/src/main/java/alluxio/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -219,8 +220,9 @@ public FileSystem load(String userKey) throws Exception {
       }
     } catch (Exception e) {
       // ignore
-      LOG.warn("Cannot create SupportedHdfsActiveSyncProvider."
-          + "HDFS ActiveSync will not be supported.");
+      LOG.warn("Cannot create SupportedHdfsActiveSyncProvider. "
+          + "HDFS ActiveSync will not be supported. "
+          + "Please upgrade to an HDFS version > 2.7 to enable support for HDFS ActiveSync");
```

Comment:
is it 2.7 or 2.6.1? Our doc says 2.6.1 https://docs.alluxio.io/os/user/stable/en/core-services/Unified-Namespace.html#active-sync-for-hdfs